server/assistanthandler.go

```diff
@@ -753,3 +854,57 @@ func unstreamResponse(rawResponse string) (*model.Message, error) {
 
 	return message, nil
 }
+
+func parseDateRangeFromQueryString(r *http.Request) (time.Time, time.Time, error) {
```

Comment:
Seems like we are adding a new date parser often. Is this a new time format that we needed specifically? I think Hunt code already knows how to parse out a date range since those arrive to the API as a range as well. Any chance of code reuse here?